### ISSUE TITLE
Source Code Files Not Up To Date With EPA Files

### ISSUE THREAD

**Issue by LRossman** [OpenWaterAnalytics/epanet-msx#7]
The source code files in this repository are out of date when compared with those published by US EPA on their [web site ](https://cfpub.epa.gov/si/si_public_file_download.cfm?p_download_id=500759&Lab=NHSRC). The latter files are from 2010/2011 while those in this repository are from 2007/2008. The newer EPA files include a fix for the "parsing exponents" problem noted in issue #1 . When comparing the two sets of files I found that 19 of 30 files had been updated in the EPA release.

**Comment by samhatchett** [comment]
indeed! thanks for catching this. I will open a new branch for that version and see how extensive the diff is. Would you be willing to review a PR and make sure we are not breaking anything with a merge?

**Comment by samhatchett** [comment]
I skimmed over the changes and agree with your assessment, @LRossman - the version bump looks like it affects the exponent bug and the JIT compilation for the most part. All other changes look to be formatting/whitespace and some error checking. The biggest issue I have with attempting a merge is the lack of a unified build configuration. CMake is great for this - so we might think about swapping in a single cmake config as a part of this update?

**Comment by LRossman** [comment]
@samhatchett I have cmake scripts ready to go (see attached files). They assume the project has sub-directories `src`, `run`, and `include`. Re-name the attached `CMakeLists_1.txt` to `CMakeLists.txt` and place it in the top-level directory. Rename `CMakeLists_2.txt` to `CMakeLists.txt` and place it in the `run `directory, along with `msxmain.c`. The `include `directory should contain `epanetmsx.h` and `epanetmsx.def`. I think it should also include `epanet2.h`, `epanet2_enums.h`, and `epanet2.lib` from the OWA-EPANET v2.2 repository.
[CMakeLists_1.txt](https://github.com/OpenWaterAnalytics/epanet-msx/files/6279513/CMakeLists_1.txt)
[CMakeLists_2.txt](https://github.com/OpenWaterAnalytics/epanet-msx/files/6279514/CMakeLists_2.txt)
